Library/Formula/cracklib.rb

```diff
@@ -22,5 +25,17 @@ def install
                           "--without-python",
                           "--with-default-dict=#{HOMEBREW_PREFIX}/share/cracklib-words"
     system "make", "install"
+
+    resource("cracklib-words").stage do |r|
+      share.install "cracklib-words-#{r.version}" => "cracklib-words"
+    end
+  end
+
+  def post_install
+    system "#{bin}/cracklib-packer < #{share}/cracklib-words"
```

Comment:
```pipe_output``` is not available in install section.

Moving code from post_install to install breaks the build.

```
==> Verifying cracklib--cracklib-words-20080507.gz checksum
==> /usr/local/Cellar/cracklib/2.9.5/bin/cracklib-packer < /usr/local/Cellar/cracklib/2.9.5/share/cracklib-words
/usr/local/share/cracklib-words.pwd: Operation not permitted
/usr/local/share/cracklib-words: Operation not permitted
/usr/local/Library/Homebrew/debrew.rb:11:in `raise'
```